core/device_hooks/cuda_hooks.cpp

```diff
@@ -109,8 +109,10 @@ void CudaExecutor::synchronize() const GKO_NOT_COMPILED(cuda);
 
 void CudaExecutor::run(const Operation &op) const
 {
+    this->template log<log::Logger::operation_launched>(this, &op);
```

Comment:
These are necessary just for completeness sake, or is there any specific reason you needed these logs here ?

Reply:
They were missing before, which caused the the tests for named Lambdas to fail.

Reply:
One reason I added this is because otherwise some tests would be failing :D